lib/sms.rb

```diff
@@ -0,0 +1,21 @@
+require 'twilio-ruby'
+
+class SMS
+
+  def send_text(message)
```

Comment:
Be careful, the sensitive data might be exposed :) (private)
Some methods may be moved to private as there is no need to call them to make the program work.